P5/Source/Specs/constraintDecl.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © TEI Consortium. Dual-licensed under CC-by and BSD2 licenses; see the file COPYING.txt for details. -->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" module="tagdocs" ident="constraintDecl">
+  <gloss versionDate="2023-03-09" xml:lang="en">constraint declaration</gloss>
+  <desc versionDate="2023-03-09" xml:lang="en">contains declarations pertaining to formal constraints expressed elsewhere in <gi>constraintSpec</gi> elements</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.encodingDescPart"/>
+    <!-- <constraintDecl> is part of content of <schemaSpec> directly, not via a class -->
+  </classes>
+  <content>
+    <sequence>
+      <alternate minOccurs="0" maxOccurs="unbounded">
+        <classRef key="model.identEquiv"/>
+        <classRef key="model.descLike"/>
+      </alternate>
+      <anyElement/>             <!-- typically <sch:ns> elements -->
```

Comment:
I would suggest making this optional, in case one only wanted to use `<constraintDecl/>` to declare a query language binding and nothing else.

Reply:
I think you are quite right.
Good catch.
Hope to make change today.

Reply:
Not just, BTW, in case user wants **no** `<sch:ns>`, but it would be appropriate to let poor user have 2 or more of them!